integration_tests/src/main/python/conftest.py

```diff
@@ -327,4 +327,35 @@ def mortgage(request):
     else:
         yield MortgageRunner(mortgage_format, mortgage_path + '/acq', mortgage_path + '/perf')
 
+class TpcdsRunner:
+  def __init__(self, tpcds_format, tpcds_path):
+    self.tpcds_format = tpcds_format
+    self.tpcds_path = tpcds_path
+    self.setup(get_spark_i_know_what_i_am_doing())
+
+  def setup(self, spark):
+    jvm_session = _get_jvm_session(spark)
+    jvm = _get_jvm(spark)
+    formats = {
+      "csv": jvm.com.nvidia.spark.rapids.tests.tpcds.TpcdsLikeSpark.setupAllCSV,
+      "parquet": jvm.com.nvidia.spark.rapids.tests.tpcds.TpcdsLikeSpark.setupAllParquet,
+      "orc": jvm.com.nvidia.spark.rapids.tests.tpcds.TpcdsLikeSpark.setupAllOrc
+    }
+    formats.get(self.tpcds_format)(jvm_session,self.tpcds_path)
```

Comment:
Worth adding a check here for an unknown format and throwing a more useful error message?  As written users are going to get something particularly unhelpful like: `TypeError: 'NoneType' object is not callable`

Reply:
I can, but this is only called from a single set of python tests where the query names are hard coded already.

Reply:
Sorry this is for the format